docs/eos.md

```diff
@@ -84,6 +92,10 @@ docker-compose exec ocis eos ls -l /eos/dockertest/reva/users/4/4c510ada-c86b-48
 -rw-r--r--   1 einstein users              10 Jul  1 15:24 newfile.txt
 ```
 
+If the upload did not work, run `docker-compose exec mgm-master eos space set default on` then try again.
```

Comment:
If the upload did not work, you can check the status of the eos space using `docker-compose exec mgm-master eos fs ls`. In case you see the `default` space marked as `offline`, run ...